Assets/Scripts/SelectionController.cs

```diff
@@ -20,13 +21,21 @@ void Start()
     {
         cubes = GameObject.FindGameObjectsWithTag("Selectable");
         numberOfCubes = cubes.Length;
+
```

Comment:
`
   var cubes = GameObject.FindGameObjectsWithTag("Selectable");
   if(cubes != null)
   {
        cubesScripts = new Selected[cubes.Length];

        for (int i = 0; i < cubes.Length; i++)
        {
            cubesScripts[i] = cubes[i].GetComponent<Selected>();
        }
   }
`